test/unit/communication/test_comm_p2p.cpp

```diff
@@ -103,21 +102,21 @@ TEST_F(P2PTestMC, SendRecv) {
   auto dist = matrix::Distribution({13, 13}, {13, 13});
 
   // single tile matrix whose columns are stored in contiguous memory
-  testSendRecv(world, MatrixT(dist, matrix::tileLayout(dist, 13, 13)));
+  testSendRecv(world, MatrixT(dist, matrix::MatrixAllocation::Tiles, matrix::Ld::Compact));
 
   // single tile matrix whose columns are stored in non-contiguous memory
-  testSendRecv(world, MatrixT(dist, matrix::colMajorLayout(dist, 13)));
+  testSendRecv(world, MatrixT(dist, matrix::MatrixAllocation::ColMajor, 26));
```

Comment:
Am I understanding this change correctly in that the previous version wasn't actually using non-contiguous memory?

Reply:
Exactly... column major layout was used but with a single contiguous tile.